tests/river/unit/extractor/extract/test_query_builder.py

```diff
@@ -82,11 +32,18 @@ def test_with_real_analysis(mimic_mapping, snapshot):
 
 
 @mock.patch("river.common.analyzer.sql_column.hashlib.sha1")
-@mock.patch("river.extractor.query_builder.Table", mock_table)
 def test_sqlalchemy_query(mock_sha1):
     mock_sha1.return_value.hexdigest.return_value = "hash"
 
     analysis = Analysis()
+    analysis.source_credentials = {
+        "host": "mimic",
+        "port": 5432,
+        "database": "test",
+        "login": "test",
+        "password": "test",
+        "model": "POSTGRES",
+    }
```

Comment:
I think mimic credentials should be a fixture which checks that the db connection with mimic can be established. Also, these variables should be picked from env variables.

Reply:
we could use this fixture in the `testy` PR to change the credentials of the mimic mapping

Reply:
I think you forgot to use your fixture here